docs/how_to_add_packages.md

```diff
@@ -154,10 +155,9 @@ Also, **every `conanfile.py` should be accompanied by one or several folder to t
 ### The test package folders: `test_package` and `test_<something>`
 
 All the packages in this repository need to be tested before they join ConanCenter. A `test_package` folder with its corresponding `conanfile.py` and
-a minimal project to test the package is strictly required. You can read about it in the
+a minimal project to test the package is strictly required. You can read about it in the [Conan documentation](https://docs.conan.io/en/latest/creating_packages/getting_started.html#the-test-package-folder).
```

Comment:
```suggestion
a minimal project to test the package is strictly required. You can read about it in the [Conan documentation](https://docs.conan.io/en/latest/creating_packages/getting_started.html).
```
That section no longer exist in docs.conan.io 😢

Reply:
I can not find any docs for this 😱